docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -1553,6 +1639,82 @@ For more details, see [GraphiteMergeTree](../../engines/table-engines/mergetree-
+| `url`                | To match the request URL, you can use the 'regex:' prefix to use regex match (optional)                                                           |
+| `empty_query_string` | Check that there is no query string in the URL                                                                                                    |
+| `methods`            | To match request methods, you can use commas to separate multiple method matches (optional)                                                       |
+| `headers`            | To match request headers, match each child element (child element name is header name), you can use 'regex:' prefix to use regex match (optional) |
+| `handler`            | The request handler                                                                                                                               |
+
+`handler` contains the following parameters:
+
+| Parameter          | Definition                                                                                                                                                            |
+|--------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `type`             | Supported types: static, dynamic_query_handler, predefined_query_handler, redirect                                                                                    | 
+| `query`            | Use with predefined_query_handler type, executes query when the handler is called                                                                                     |
+| `query_param_name` | Use with dynamic_query_handler type, extracts and executes the value corresponding to the <query_param_name> value in HTTP request params                             |
+| `status`           | Use with static type, response status code                                                                                                                            |
+| `content_type`     | Use with static type, response content-type                                                                                                                           |
+| `response_content` | Use with static type, Response content sent to client, when using the prefix 'file://' or 'config://', find the content from the file or configuration send to client |
+| `url`              | A location for redirect                                                                                                                                               |
+
+
+Along with a list of rules, you can specify `<defaults/>` which specifies to enable all the default handlers.
+
+You will find the following example, commented out, in config.xml:
```

Comment:
Let's replace this by just "Example:".

Reasons:
1. "config.xml" is not specific enough, I don't even know what directory it is in when ClickHouse is installed via installer or package manager.
2. I honestly don't like the fact that we ship config.xml at all, perhaps we will stop doing so at some point. The primary source of documentation should be this end-user markdown file, not some obscure XML document.